packages/client/src/Config.ts

```diff
@@ -141,7 +141,7 @@ export const STREAM_CLIENT_DEFAULTS: StrictStreamrClientConfig = {
 
     // Encryption options
     verifySignatures: 'auto',
-    groupKeys: {}, // {streamId: groupKey}
+    encryptionKeys: {}, // {streamId: groupKey}
```

Comment:
Maybe remove comment?
```suggestion
    encryptionKeys: {}, // {streamId: groupKey}
```